publication/paper.md

```diff
@@ -47,21 +47,66 @@ bibliography: paper.bib
+In contrast, archival tags store sensor measurements at set intervals in their memory.
+To access the logged data, these tags must either be recovered (which mostly depends on fishers and the local population living along the coast) or transmit their information via satellite. The data from archival tags can offer detailed insights into vertical movement patterns [@heerah2017coupling], environmental preferences [@righton2010thermal; @skagerrak_tagging], and can be used to reconstruct migration paths through geolocation modeling.
+
+Over the past two decades, several geolocation methods have been developed to track fish movements.
+The first approach compares tag records with field observations of variables like light intensity and temperature, assuming fixed swimming speeds [@metcalfe1997tracking; hunter2003geolocation].
+A more advanced method uses state-space models (SSMs), which are flexible statistical tools that estimate the likelihood of unknown states, such as fish location and behavior [@royer2005state; @patterson2017statistical].
+SSMs can account for uncertainties in both tag records and field observations.
+
+Hidden Markov Models (HMMs) are a specific type of SSM used for geolocation.
+HMMs represent fish locations as discrete states within a spatial grid and consist of two sub-models: the process model, which predicts movement between grid cells, and the observation model, which predicts the likelihood of observations based on the state. The estimation process involves iterative "time updates" and "data updates" to refine the fish's location over time, culminating in a "forward filter" and a "backward filter" to incorporate all available data [@pedersen2008geolocation; @woillez_hmm-based_2016].
+
+![Example of an acoustic tag (on the left) and a DST (on the right). See the centimeter scale for size reference.\label{fig:tag}](archival_tag.png){ width=35% }
+
+In practice, estimating fish positions relies on the likelihood of observed data from archival tag logs, such as temperature at specific depths, combined with reference geoscience data like satellite observations and oceanic temperature from hydrodynamic models. The quality of these geophysical reference fields is crucial and depends on the spatio-temporal resolution at which physical processes are modeled. A first approach to enhance the accuracy of geolocation model predictions involved using additional information.
+Acoustic tags provide location data when tagged animals are within range of a receiver, while archival tags offer continuous time series of logged sensor measurements, from which trajectories can be inferred.
+These technologies provide complementary information on fish movement patterns, helping to overcome the limitations of each method. Fish trajectories can be reconstructed from logged depth and temperature histories using an existing geolocation modeling approach, adapted to include the likelihood of the acoustic detections [@a_combination_tag_2023; @100008].
```

Comment:
@100008 why?

Reply:
FYI you tagged someone with this... to avoid that, you can enclose the reference in backticks